src/page/product-added-modal.page.ts

```diff
@@ -5,9 +5,10 @@ export class ProductAddedPage {
   private timeout: number = 5000;
 
   constructor () {
-    this.addToCartBtn = $('#center_column a.button.ajax_add_to_cart_button.btn.btn-default');
+    // in this way we can select another product in the list by modifying the nth-child
+    // also, a.button.ajax_add_to_cart_button is enough
+    this.addToCartBtn = $('#center_column ul li:nth-child(1) a.button.ajax_add_to_cart_button');
```

Comment:
I just figure out, you confused product-added-modal with product-list page objects. This should have the proceedToCheckout